lisa/tools/windows_feature.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.tools.powershell import PowerShell
+from lisa.util import LisaException
+
+
+# WindowsFeature management tool for Windows Servers.
+# It can install, uninstall, and check the status of Windows features.
+# Hyper-V, DHCP etc. are examples of Windows features.
+# This tool uses PowerShell to manage Windows features.
+# Not supported on PC versions like Windows 10, 11 etc.
+class WindowsFeature(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        try:
+            self.node.tools[PowerShell].run_cmdlet(
+                "Get-WindowsFeature",
+                force_run=True,
+            )
+            self._log.debug("'Get-WindowsFeature' is installed")
+            return True
+        except LisaException as e:
+            self._log.debug(f"'Get-WindowsFeature' is not available: {e}")
+            return False
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._powershell = self.node.tools[PowerShell]
+
+    def install_feature(self, name: str) -> None:
```

Comment:
`install`

Reply:
This is feature installation not tool installation.